lib/bot.js

```diff
@@ -170,15 +170,22 @@ class Bot {
   sendToDiscord(author, channel, text) {
     const discordChannelName = this.invertedMapping[channel.toLowerCase()];
     if (discordChannelName) {
-      // #channel -> channel before retrieving:
-      const discordChannel = this.discord.channels.find('name', discordChannelName.slice(1));
+      // #channel -> channel before retrieving and select only text channels:
+      const discordChannel = this.discord.channels.find(
+        (o) => o.name === discordChannelName.slice(1) && o.type === 'text'
```

Comment:
You'll have to remove the parenthesis around `(o)` to pass the linter.
Might be cleaner to write it something like this though, maybe?
```js
this.discord.channels
  .filter(c => c.type === 'text')
  .find('name', discordChannelName.slice(1));
```